### DISCUSSION TITLE
Cannot get -Dhttp.port to work in 2.9.0-RC3

### DISCUSSION THREAD

**Discussion by nilsweinander** [playframework/playframework#12050] [Q&A] [answered]
I have just created a new Play project, using 2.9.0-RC3. In previous versions I have used (2.3 and 2.7), I have been able to change the port the app runs on when I start it from IntelliJ, by specifying

-Dhttp.port=XXXX

in JVM options. In the new 2.9 project this has no effect,. Neither has setting

play.server.http.port=XXXX

in application.conf,  Play uses port 9000 regardless. What am I doing wrong here?

**Comment by PromanSEW** [comment, marked as answer]
If you start app from IDEA, you likely do it in dev mode. In this case you should set port in `build.sbt`:
```scala
PlayKeys.devSettings += ("play.server.http.port", "XXXX")
```

**Reply by nilsweinander** [reply]
Thanks! That did the trick.
